_includes/layout/header-scripts.html

```diff
@@ -27,7 +27,7 @@
     {
       label: "PWA",
       name: "pwa-devdocs",
-      baseUrl: "https://magento-research.github.io/pwa-studio"
+      baseUrl: "https://magento.github.io/pwa-studio"
```

Comment:
```suggestion
      baseUrl: "https://pwastudio.io"
```
it looks like you were right, eric must have changed the behavior.
this can be changed to: `pwastudio.io` and it will redirect correctly: https://pwastudio.io/peregrine/reference/hooks/useQuery/

Reply:
I think it's better to avoid redirects than have a pretty url in the search. It's not a big deal, but i just dont think redirecting is a good idea on a regular basis.